src/components/shared/input/input.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./input.scss";
+
+const TextField = () => (
```

Comment:
don't look like shared because you define by class that this input is for form.
as an option you can pass the class from parent so it can be reusable.

Reply:
Done